src/components/guide/Tab.tsx

```diff
@@ -0,0 +1,18 @@
+import { TabsTrigger } from "@radix-ui/react-tabs";
+import { ReactNode } from "react";
```

Comment:
```suggestion
import type { ReactNode } from "react";
```

the new standard for importing types (not functions, classes etc)